packages/shared/src/utils/helpers/details.ts

```diff
@@ -70,6 +70,26 @@ export const getDurationUnitText = (obj_duration: moment.Duration) => {
     return unit_map.s.name;
 };
 
+export const getTurbosDurationText = (obj_duration: moment.Duration) => {
+    const unit_map = getUnitMap();
+    const duration_ms = obj_duration.asMilliseconds() / 1000;
+
+    if (duration_ms >= 86400000) {
```

Comment:
@maryia-deriv Would you please move these magic numbers into named variables for better readability 🙏🏻 🙇🏻

Reply:
@farzin-deriv we got rid of this function, updated an existing getDurationUnitText() function to suit our purpose instead where milliseconds are used directly since they are just milliseconds and the name of the function speaks for itself.🙂